react-notes/src/components/overlayingAlert/overlayingAlert.jsx

```diff
@@ -0,0 +1,25 @@
+import Alert from '@material-ui/lab/Alert';
+import { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+const OverlayingAlert = ({ children, status }) => {
+  const [open, setOpen] = useState(false);

```

Comment:
What is open? 
Use please `isOpen` instead. The same for the future.
```suggestion
  const [isOpen, setIsOpen] = useState(false);
```